pseudocontainers_task/student/include/impl.h

```diff
@@ -16,4 +18,28 @@ constexpr char vacationMarker = 'x';
  * @param workerVacations - pairs of worker and his calendar
  * @return Max intersection of two workers and intersection days count
  */
-std::tuple<worker_id, worker_id, days> getMaxIntersection(const std::unordered_map<worker_id, calendar>& workerVacations);
+std::tuple<worker_id, worker_id, days> getMaxIntersection(const std::unordered_map<worker_id, calendar>& workerVacations){
+    days max_intersection = 0;
+    worker_id worker1, worker2;
+
+    for( auto worker: workerVacations){
```

Comment:
use const ref by default please
you should use auto without reference only when you're dealing with literal types